components/issue-proposal-progress-bar.tsx

```diff
@@ -28,7 +28,7 @@ export default function IssueProposalProgressBar() {
   const isFinalized = !!networkIssue?.closed;
   const isInValidation = !!networkIssue?.isInValidation;
   const isIssueinDraft = !!networkIssue?.isDraft;
-  const creationDate = networkIssue?.creationDate;
+  const creationDate = networkIssue?.creationDate || new Date();
```

Comment:
it's a bit impossible for creationDate to not exist, but if it doesnt: It shouldn't be "todays date"